### PR TITLE
Hotfix for external IDs on the Project Grid

### PR DESCRIPTION
**Pull request by illusional**
Views like this:

image

### CODE REVIEW

File: web/src/pages/project/ProjectGrid.tsx [hunk 1]
```diff
@@ -300,9 +306,7 @@ const ProjectGrid: React.FunctionComponent<ProjectGridProps> = ({
                                 // const border = '1px solid'
                                 // debugger
                                 return (
-                                    <SUITable.Row
-                                        key={`${p.external_id}-${s.id}-${sg.id}-${assay.id}`}
-                                    >
+                                    <SUITable.Row key={`${p.id}-${s.id}-${sg.id}-${assay.id}`}>
```
Comment (jmarshall) [inline web/src/pages/project/ProjectGrid.tsx:309]: Does this need any corresponding updates elsewhere (e.g. in code that tries to link to this row key)?

Comment (illusional) [reply web/src/pages/project/ProjectGrid.tsx:309]: Na, [keys](https://legacy.reactjs.org/docs/lists-and-keys.html#keys) are just for react to tell what's changed between renders

Comment (jmarshall) [reply web/src/pages/project/ProjectGrid.tsx:309]: Excellent.

**Review by jmarshall — APPROVED** [review]
LGTM. I can think of maybe slightly less gross ways to do it (e.g. rename `sanitiseValue` to `displayValue`, maybe give it `k` as a parameter too, and teach it how to display `external_ids` specially; or alternatively have `formatExternalIds` able to format single ids too) but this is great as a hot fix.

**Comment by illusional** [comment]
Yeah agreed :) I've completely rewritten the page in #742, so no point doing anything super crazy here

**Pull request merged**